Lib/gzip.py

```diff
@@ -17,6 +17,14 @@
 
 READ, WRITE = 1, 2
 
+# OS field values as specified in RFC 1952
+OS_CODES = {
+    'linux': 3,
+    'os2': 6,
+    'win32': 11,
```

Comment:
Actually two most common file systems on Windows are FAT and NTFS.

Reply:
Looking at https://docs.python.org/3/library/sys.html we ave the following available:
- aix
- linux
- win32
- cygwin
- darwin

Reply:
Fixed in 851c64b995b454e98d541236f48724bf2b59e3b2